src/grpcd/runtime.rs

```diff
@@ -1053,6 +1112,22 @@ impl Farcaster for FarcasterService {
                     funding_infos: infos
                         .swaps_need_funding
                         .iter()
+                        .filter(|info| {
+                            if network_selector == NetworkSelector::AllNetworks {
+                                true
+                            } else {
+                                match info {
```

Comment:
Implement this logic on `FundingInfo` struct directly instead

Reply:
I don't quite get what you want here.

Reply:
I think what I had in mind is wrong, this can stay as is.